xradar/io/export/odim.py

```diff
@@ -145,12 +145,17 @@ def to_odim(dtree, filename, compression="gzip", compression_opts=6):
 
     Keyword Arguments
     -----------------
+    source : str
+        radar identifier (see ODIM documentation)
     compression : str
         Compression filter name, defaults to "gzip".
     compression_opts : compression strategy
         options as needed by above filter, defaults to 6
-
     """
+
+    if source is None:
```

Comment:
Here we might add that check, if at least one of those descriptors is given with `source`.